tests/mock/AnnotatorPageTest.php

```diff
@@ -1,19 +1,28 @@
-<?php
-
-
-class AnnotatorPageTest extends Page implements TestOnly
-{
-    private static $db = array(
-        'SubTitle'    => 'Varchar(255)'
-    );
-}
-
-class AnnotatorPageTest_Controller extends Page_Controller implements TestOnly
-{
-
-}
-
-class AnnotatorPageTest_Extension extends Extension implements TestOnly
-{
-
-}
+<?php
+
+namespace Axyr\IDEAnnotator\Tests;
+
+require_once('/mysite/code/Page.php');
+require_once('/mysite/code/PageController.php');
+
+use Page;
+use PageController;
+use SilverStripe\Core\Extension;
+use SilverStripe\Dev\TestOnly;
+
+class AnnotatorPageTest extends Page implements TestOnly
+{
+    private static $db = array(
+        'SubTitle'    => 'Varchar(255)'
+    );
+}
+
+class AnnotatorPageTestController extends PageController implements TestOnly

```

Comment:
We need to separate these into their own files

Reply:
Yeps, all old SS3 stuff still